peps/pep-0811.rst

```diff
@@ -0,0 +1,322 @@
+  Examples of disallowed behavior include, sharing information with colleagues
+  or publicly deploying unpublished mitigations or patches ahead of the advisory
+  publication date.
+* Acting as a "Coordinator" of vulnerability reports that are submitted
+  to projects. Coordinators responsibility is to move a report through the PSRT
+  process to a "finished" state, either rejected or as a published advisory and
+  mitigation, within the industry standard timeline of 90 days.
+* As a Coordinator, involving relevant core team members or triagers where
+  necessary to make a determination whether a report is a vulnerability and
+  developing a patch. Coordinators are **encouraged** to involve members of
+  the core team to make the best decision for each report rather than working
+  in isolation.
+* As a Coordinator, calculating the severity using CVSS and authoring advisories
+  to be shared on `security-announce@python.org`_. These advisories are used
+  for CVE records by the PSF CVE Numbering Authority.
+* Coordinators that can no longer move a report forwards for any reason must
+  delegate their Coordinator role to someone else in the PSRT.
+* PSRT members that are admins will have additional responsibilities.
+* PSRT members that are staff of the Python Software Foundation, as an
+  "Open Source Steward" defined in `Article 24 of the Cyber Resilience Act`_,
+  have `additional responsibilities`_, such as reporting actively exploited
+  vulnerabilities to ENISA/CSIRTs.
+
+.. _security-announce@python.org: https://mail.python.org/archives/list/security-announce@python.org/
+.. _Article 24 of the Cyber Resilience Act: https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=OJ:L_202402847#art_24
+.. _additional responsibilities: #responsibilities-of-psf-staff-psrt-members
+.. _Python Developer Guide: https://devguide.python.org/developer-workflow/psrt/
+
+Responsibilities of PSRT Admins
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+PSRT members that are designated as admins by the Steering Council have the
+following additional responsibilities:
+
+* Managing the GitHub team, mailing list, Discord channel, and other
+  PSRT venues to ensure they are synchronized with the canonical list of
+  PSRT members determined by the Steering Council.
+* On a yearly basis, providing the Steering Council with a report including
+  a list of inactive PSRT members.
+
+Responsibilities of PSF Staff PSRT members
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Python Software Foundation acts as the "Open Source Steward" for
+CPython, pip, and other projects according to the Cyber Resilience Act (CRA).
+Therefore, vulnerability reporting has additional requirements for PSF staff
+detailed in CRA `Article 24 <https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=OJ:L_202402847#art_24>`_.
+These requirements can be summarized as:
+
+➤ Maintain a vulnerability disclosure policy fostering the voluntary reporting of vulnerabilities.
```

Comment:
Use standard bullet points for consistency and accessibility (here and below):
```suggestion
* Maintain a vulnerability disclosure policy fostering the voluntary reporting of vulnerabilities.
```
The following lines might need indent adjusted too.